python/tests/test_petab_objective.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for petab_objective.py."""
+
+from pathlib import Path
+
+import amici
+import amici.petab_import
+import amici.petab_objective
+import petab
+import pytest
+
+
+# Absolute and relative tolerances for finite difference gradient checks.
+ATOL: float = 1e-3
+RTOL: float = 1e-3
+
+
+@pytest.fixture
+def lotka_volterra() -> petab.Problem:
+    return petab.Problem.from_yaml(str(
+        Path(__file__).parent
+        / 'petab_test_problems'
+        / 'lotka_volterra'
+        / 'petab'
+        / 'problem.yaml'
+    ))
+
+
+def test_simulate_petab_sensitivities(lotka_volterra):
+    petab_problem = lotka_volterra
+    amici_model = amici.petab_import.import_petab_problem(petab_problem)
+    amici_solver = amici_model.getSolver()
+
+    amici_solver.setSensitivityOrder(amici.SensitivityOrder_first)
+
+    result = amici.petab_objective.check_grad_multi_eps(
+        petab_problem=petab_problem,
+        amici_model=amici_model,
+        amici_solver=amici_solver,
+        detailed=True,
+    )
+
+    assert (result.abs_err < ATOL).all()
```

Comment:
we discussed this, right? seems plausible to me this way. if problematic, maybe use not the optimal parameter vector.

Reply:
We discussed something similar in pyPESTO [1], where an optimized vector is used, and the value of `RTOL` is high (`1`) (especially given an optimized vector).

Here, `RTOL` is smaller (`1e-3`), and an optimized vector is not used (also, the measurements here are nonsense).

[1] https://github.com/ICB-DCM/pyPESTO/blob/main/test/petab/test_amici_objective.py#L116

Reply:
sounds good then